### PR TITLE
Make SDK compliant with CloudEvents SDK spec

### PR DESCRIPTION
**Pull request by denismakogon**
SDK spec was merged.
Now we need to make all SDK compliant with it.
So, as for now, "upstream" becomes "v02" and the rest of API methods are now compliant.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,12 +25,12 @@ from cloudevents.sdk.event import v01
...
     v01.Event().
-    WithContentType("application/json").
-    WithData('{"name":"john"}').
-    WithEventID("my-id").
-    WithSource("from-galaxy-far-far-away").
-    WithEventTime("tomorrow").
-    WithEventType("cloudevent.greet.you")
+    SetContentType("application/json").
+    SetData('{"name":"john"}').
+    SetEventID("my-id").
```
Comment (duglin) [inline README.md]: any reason this isn't just `SetID` to align with the property name change? Same for Time and Type below.

Comment (denismakogon) [reply README.md]: That’s what’s in SDK spec. Verbose API is a better option here, less things seem to be ambiguous.

**Comment by denismakogon** [comment]
Just need a formal approval to release a new version of an SDK.

**Comment by duglin** [comment]
LGTM

**Pull request merged**